src/guacd/daemon.c

```diff
@@ -257,6 +272,17 @@ static void guacd_handle_connection(guacd_client_map* map, guac_socket* socket)
             sizeof(char*) * video->argc);
     client->info.video_mimetypes[video->argc] = NULL;
 
+    /* Handle image mimetypes */
+    for (int i = 0; i < image->argc; i++) {
+        /* Check if webp image encoding is supported by client */
+        if (strcmp(image->argv[i], "image/webp") == 0) {
+            client->info.supports_webp = 1;
```

Comment:
Including a special-purpose flag veers from the pattern we already use for determining audio and video codec support.

Why not use the same array of mimetypes mechanism?